.github/workflows/config/spelling_allowlist.txt

```diff
@@ -13,6 +15,7 @@ CPU
 CPUs
 CUDA
 CentOS
+Classcial
```

Comment:
Please do not greenlist words here without checking that they can be found in the dictionary. I assume this was meant to say "classical"? The spellchecker correctly flags this, and it needs to be removed here.